test/tests/functional/pbs_qrun.py

```diff
@@ -129,6 +129,7 @@ def test_qrun_hangs(self):
             try:
                 self.server.runjob(jobid=jid)
                 self.logger.info("Successfully runjob. Child process exit.")
+                os._exit(0)
```

Comment:
@saimanikant I think you raised PR in wrong branch?

Reply:
THanks @hirenvadalia